src/main/java/com/example/Trello/controllers/CardController.java

```diff
@@ -0,0 +1,53 @@
+package com.example.Trello.controllers;
+
+import com.example.Trello.model.dto.card.Card;
+import com.example.Trello.model.dto.card.CardCreation;
+import com.example.Trello.services.CardService;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/boards/{board_id}")
```

Comment:
It's a separate entity, you don't need nested paths.
```suggestion
@RequestMapping("/cards")
```

Reply:
there's a button `Commit suggestion`, don't use it. Change code always in your IDEA